src/components/calcite-input/calcite-input.e2e.ts

```diff
@@ -808,5 +808,77 @@ describe("calcite-input", () => {
       expect(await calciteInput.getProperty("value")).toBeFalsy();
       expect(await input.getProperty("value")).toBeFalsy();
     });
+
+    it(`allows setting value to undefined after initial load.`, async () => {
+      const initialValue = "1234";
+      const page = await newE2EPage({
+        html: `<calcite-input type="number" value="${initialValue}"></calcite-input>`
+      });
+      const calciteInput = await page.find("calcite-input");
+      const input = await page.find("input");
+
+      expect(await calciteInput.getProperty("value")).toBe(initialValue);
+      expect(await input.getProperty("value")).toBe(initialValue);
+
+      calciteInput.setProperty("value", undefined);
+      await page.waitForChanges();
+
+      expect(await calciteInput.getProperty("value")).toBeFalsy();
+      expect(await input.getProperty("value")).toBeFalsy();
+    });
+
+    it(`disallows setting value to "undefined" after initial load.`, async () => {
+      const initialValue = "1234";
+      const page = await newE2EPage({
+        html: `<calcite-input type="number" value="${initialValue}"></calcite-input>`
+      });
+      const calciteInput = await page.find("calcite-input");
+      const input = await page.find("input");
+
+      expect(await calciteInput.getProperty("value")).toBe(initialValue);
+      expect(await input.getProperty("value")).toBe(initialValue);
+
+      calciteInput.setProperty("value", "undefined");
+      await page.waitForChanges();
+
+      expect(await calciteInput.getProperty("value")).toBe(initialValue);
+      expect(await input.getProperty("value")).toBe(initialValue);
+    });
+
+    it(`disallows setting value to "null" after initial load.`, async () => {
+      const initialValue = "1234";
+      const page = await newE2EPage({
+        html: `<calcite-input type="number" value="${initialValue}"></calcite-input>`
+      });
+      const calciteInput = await page.find("calcite-input");
+      const input = await page.find("input");
+
+      expect(await calciteInput.getProperty("value")).toBe(initialValue);
+      expect(await input.getProperty("value")).toBe(initialValue);
+
+      calciteInput.setProperty("value", "null");
```

Comment:
is this actually setting to `null` or the string `"null"`?

Reply:
the string "null".  These tests are kinda redundant, but wanted to include them anyway.  There's an actual test for the `null` value also in there, same for undefined.

Reply:
What is this testing exactly? From the component's perspective, it's just receiving an invalid number string, right? If so, I'd suggest removing it since there's another test that would catch this case.